### DISCUSSION TITLE
Convert a JSON response to UTF-8

### DISCUSSION THREAD

**Discussion by Kolhun** [curl/curl#11726] [Q&A] [answered]
Good afternoon, I'm working with curl and the question arose how to get data in Cyrillic? In the screenshot, the answer contains Cyrillic and I get the code of the characters themselves, how can this be corrected?

![image](https://github.com/curl/curl/assets/36815965/05d8db34-b658-4b64-882b-63f69b2a9ca6)

**Comment by bagder** [comment, marked as answer]
This is not really a curl problem since curl only delivers exactly what the server sends you.

An easy way way to fix your problem could be to pipe it through `jq`:

~~~
$ cat json
{
 "name":"\u0410\u0440\u0430"
}
$ cat json | jq
{
  "name": "Ара"
}

**Reply by Kolhun** [reply]
is jq a third-party library? Or how?

**Reply by bagder** [reply]
You actually could not just search for that? It is a command line tool.

https://jqlang.github.io/jq/

**Reply by Kolhun** [reply]
It doesn't look like a very working option

![image](https://github.com/curl/curl/assets/36815965/63e4e5a7-aa13-45ca-b9b1-2b3538b99e75)

**Reply by bagder** [reply]
I don't understand what you're showing. I showed you my test and that worked for me.

This is not a curl question. curl worked fine. Maybe this is not the best place to get answers to how you best convert JSON.

**Reply by Kolhun** [reply]
Thank you
